src/css/main.css

```diff
@@ -736,6 +743,10 @@ a.help-link:hover {
 
 /* This is a temporaly hack, we should style the editor instead of overwriting
    the a-scene to fix the "display: block" issue. */
-a-scene {
+.aframe-inspector-opened a-scene {
```

Comment:
We should keep track of that yep, but right now there's no support for multiple scenes on the rest of the code as we're looking for example for `a-scene` in query selector. So we should do another PR to achieve that, and also expose the other scenes on the inspector UI probably so you could switch them.